examples/utils/executable_for_notebook_runtimes.py

```diff
@@ -0,0 +1,54 @@
+environment_variables_config = {'ACCOUNT_ID': ACCOUNT_ID,
```

Comment:
I'd add a comment at the top of the file explaining what its purpose is and where `ACCOUNT_ID` is defined, for example